pkg/policy/api/rule.go

```diff
@@ -45,6 +45,16 @@ type DefaultDenyConfig struct {
 	Egress *bool `json:"egress,omitempty"`
 }
 
+// LogConfig specifies custom policy-specific Hubble logging configration.
```

Comment:
Spelling nit:

```suggestion
// LogConfig specifies custom policy-specific Hubble logging configuration.
```

(and in the generated CRD YAML)